forEach.js

```diff
@@ -0,0 +1,16 @@
+let family = ["Sofi","Hen", "Len"];
+
+function forEch(arr, callback){
```

Comment:
1. Typo
2. Use custom name for custom methods

```suggestion
function customForEach(arr, callback){
```